### PR TITLE
📖 Document derived files

### PR DESCRIPTION
**Pull request by MikeSpreitzer**
## Summary
This PR updates the documentation on packaging and delivery to cover the generated sources, tracking #1608 and earlier development.

## Related issue(s)

Fixes #

### CODE REVIEW

File: docs/content/v0.20/packaging.md [hunk 1]
```diff
@@ -10,6 +10,22 @@ There is also a container image involved.
...
+- `make manifests` generates the CustomeResourceDefinition files.
+
+- `make generated` generates the deep copy code.
+
+Both of the above are currently broken, see Issue 1611.
+
+#### Files generated by code-generator
+
+The files in [pkg/generated](../../../pkg/generated) are generated by [k/code-generator](https://github.com/kubernetes/code-generator). This generation is done at development time by the command `make codegenclients`. That command does only two things: (1) ensure the needed local copy of `code-generator`, and then (2) invoke [hack/update-codegen-clients.sh](../../../hack/update-codegen-clients.sh).
+
```
Comment (ezrasilvera) [inline docs/content/v0.20/packaging.md:28]: I don't mind having this - but I haven't see someone documenting the fact that they have client-go generated code.

Comment (MikeSpreitzer) [reply docs/content/v0.20/packaging.md:28]: I have occasionally seen this documented. It is something that some contributors (the ones who add/change/delete stuff in the API) have to understand, so it should be documented.

**Comment by MikeSpreitzer** [comment]
@pdettori : the material I wrote here about files generated by controller-gen could use some elaboration, I do not understand it precisely. I see that `make manifests` does more than invoke controller-gen. I am not sure the comments on those make targets are entirely accurate.

**Pull request merged**